app/assets/stylesheets/components/_btn.scss

```diff
@@ -8,7 +8,7 @@
 }
 
 .usa-button:disabled.usa-button--active,
-[aria-disabled='true'].usa-button--active {
+.usa-button[aria-disabled='true'].usa-button--active {
```

Comment:
Without this change, the styles aren't strong enough to override default USWDS as expected, resulting in a grey button.

Before|After
---|---
![image](https://github.com/user-attachments/assets/1d85c883-a04a-42cf-9a91-347b38f9835a)|![image](https://github.com/user-attachments/assets/381ce161-88f7-4945-8edc-fde0a7fb1862)